### DISCUSSION TITLE
Automatic mails go to spam-folder

### DISCUSSION THREAD

**Discussion by dfeistauer** [rubenarslan/formr.org#581] [Q&A]
Hi,
I have a survey where participants receive an email every week to participate in the next part of the survey.
The email is sent out without any problems. The mails go out in groups of about 20 participants at a time.

However, for many participants the email ends up in the spam folder. 
Is there any way to prevent this? Not many subject check their spam folder by default.

cheers 
Daniela

**Comment by rubenarslan** [comment]
Avoiding being sent to spam is more of an art than a science (see [link](https://sendgrid.com/en-us/blog/10-tips-to-keep-email-out-of-the-spam-folder)). It can happen to you if an participant clicked report spam on your email, so it helps if you have an unsubscribe link in your email instead (send them to YOURRUNLINK/settings).
If your mail server is not properly configured (SPF, DKIM) this can also happen. If you have so few participants, this seems more likely than that someone marked you as spam. You can forward an email that got sent to spam to your university IT department to whoever is in charge of email, and they might be able to tell where the problem is. Ideally, you send password-authenticated emails in formr.

**Reply by dfeistauer** [reply]
Hi Ruben,

thanks for the fast reply.

I have an unsubcribe link in my automatic mail. The link leads to the last module in my run.
Students didn't click to report as spam. 

> Avoiding being sent to spam is more of an art than a science (see [link](https://sendgrid.com/en-us/blog/10-tips-to-keep-email-out-of-the-spam-folder))

How can I use these tips in formr? I don't see much opportunity to make these chances in formr.
I have this settings in the mail-account
![image](https://github.com/user-attachments/assets/b4f24683-03d0-4653-bd29-fa1f8f36334e)

**Reply by rubenarslan** [reply]
You can't do much about this in formr. formr simply sends the emails to an SMTP server. Potentially this SMTP server is misconfigured, so it looks unsafe/spammy. My best recommendation is

> You can forward an email that got sent to spam to your university IT department to whoever is in charge of email, and they might be able to tell where the problem is. 

You could also paste the full headers from such an email here and I can take a look. But I can't make any changes.
